src/thematic/0_generic/sfp-001_vervanging-terrest_nl/07_stappenplan.Rmd

```diff
@@ -0,0 +1,117 @@
+# Werkwijze
+
+## Uitvoering
+
+<!--
+Gedetailleerde omschrijving van alle stappen die doorlopen moeten worden om het veldprotocol uit te voeren. Subtitels gebruiken om elke stap te omschrijven. 
+-->
+
+### Voorbereiding van het veldwerk
+
+Bij voorkeur maak je gebruik van een veldcomputer met GIS-omgeving waarin je volgende GIS-lagen invoert:
+
++ de steekproefeenheden van het meetnet in kwestie;
++ de polygonen uit de habitatkaart die steekproefeenheden bevatten;
++ de rastercellen met bijhorend rangnummer van de GRTS-master steekproef die in bovenstaande polygonen gelegen zijn, exclusief de rastercellen die al zijn geselecteerd voor het meetnet (dit zijn de potentiële vervangpunten)
+
+Binnen elke polygoon ken je dan alle mogelijke vervangpunten een relatief rangnummer toe (startend van 1) op basis van het (absolute) rangnummer. Zo heb je op het terrein een beter zicht op de volgorde van de punten.Bijkomend kan er een script aangemaakt worden waarmee alle mogelijke vervangpunten voor een bepaalde steekproefeenheid kan geselecteerd worden. Op basis hiervan kan je dan makkelijker op terrein de mogelijke vervangpunten visualiseren.
+
+Als alternatief kan er gewerkt worden met afgedrukte kaarten.
+
+### Start van de vervangingsprocedure op terrein
+
+Je zoekt enkel een vervangpunt voor een steekproefeenheid als de steekproefeenheid niet geschikt is zoals gespecificeerd in het projectprotocol. 
```

Comment:
> projectprotocol

Of meer specifiek: unit-selection protocol. Dit kan een generiek (thematisch) of een projectprotocol zijn, maar dat sowieso op zijn beurt een subprotocol is van een overkoepelend projectprotocol bv. voor 1) MHQ, 2) MNE-groundwater, 3) MNE-surfacewater, enzovoort.

Het unit-selection protocol dient te verwijzen naar dit en een aquatisch vervangprotocol (andere: ihb voor aquatisch/3260/7220, waar GRTS-toevoeging en level-constrained vervangen alternatieven zijn). 
Het unit-selection protocol zal idealiter zowel terrestrische als aquatische types omvatten, vermits het MNM-terreinwerk voor beide samen kan gelden (bv. peilbuizen).

Reply:
Lijkt me een duidelijk protocol met een op het eerste zicht haalbare praktische uitvoering Toon! Figuren maken het inderdaad zeer verstaanbaar voor de lezer
Verder kijkend  naar implementatie zijn me nog een aantal zaken onduidelijk:
- hoe al die bijkomende data op het terrein kan geraadpleegd worden?
- herkartering op terrein van de doelpopulatie dient opgeslagen te worden?
- hoe gaan we al die bijkomende data beheren (elke bezocht vervangpunt dient gescoord te worden, eventuele herbezoeken in geval tijdelijke ontoegankelijkheid etc.)

Er moet hierbij gestreefd worden om zeker niet naar extra hardware te gaan, maw de zaken moeten in Esri Collector (of hoogstens bijkomende apps) kunnen gebeuren. Mogelijk vijgen voor Pasen, maar misschien toch relevant om rekening mee te houden bij de verder uitwerking, bv. heb ik er geen zicht op wat voor en hoeveel rasterpuntbestanden er allemaal nodig zullen zijn en of Collector dat aankan.

Alvast deze zaken nog :

Punt 3.3: minder geschikt voor habitattypen met grote polygonen (opp): begrijp ik maar impliceert dit dan dat er een bijkomende vervangmethode zal ontwikkeld worden? ik ben daar geen voorstander van ifv complexiteit protocols. Is het geen optie om grote polygonen op te slicen zodat er binnen deelpolygonen op dezelfde wijze kan gewerkt worden. Dit dan welliswaar onder de motorkap of met een bijkomende raster dat de bovengrenzen van grote polygonen aangeeft?
Bij  bossen lijkt mij overzichtelijkheid minder een probleem, dan wel hetzelfde als hierboven namelijk dat het vaak grotere polygonen zijn. Voor sommige moerastypes en bv. wilgenvloedbos is toegankelijkheid potentieel lastig, maar kan er dan niet de redenering gevolgd worden dat deze punten wegvallen, cfr. 7.1.5.4

Punt 7: werkwijze:

Algemeen:  een deel van het protocol zal afhankelijk zijn van de gehanteerd veldhardware en -software lijkt me

Punt 7.1.1: gis-lagen die je mee hebt: enkel rastercellen met bijhorende GRTS-ranking gelegen in originele polygonen? Hoe ga je om met vlekken doelpopulatie die de grenzen van de originele polygoon overschrijden? Die vallen per definitie buiten de vervangprocedure. Ook nabijgelegen losse habitatvlekken die buiten de originele polygoon vallen zijn dus niet mee te nemen als vervangpunten? Is het geen optie om simpelweg te werken met één GRTS-centroïden bestand voor alle types en over het volledige grondgebied? 

ook hier nadenken over automatisatie van het toekennen van het relatief rangnummer eventueel. mogelijks is dit wat je bedoelt met een script aanmaken, ook hier ongetwijfeld beperkingen van de veldsoftware mee te nemen

Issue rond cel-gebaseerde types kan ik nog niet op ingaan, want de achtergrond heb ik nog niet doorgenomen.

Tot zover voor nu, ik duik nog wat dieper in de manual 'commenting on pull request' en geef dan als dat lukt wat line comments (fingers crossed)

thanks!